pydo/models.py

```diff
@@ -74,41 +80,38 @@ class Task(Base):
     fun = Column(Integer, doc='Task fun')
     project = relationship('Project', back_populates='tasks')
 
+    parent_id = Column(String, ForeignKey('task.id'))
+    parent = relationship('Task', remote_side=[id], backref='children')
+
+    type = Column(
+        String,
+        nullable=False,
+        doc='Task type: {}'.format(str(possible_task_types))
+    )
+    __mapper_args__ = {
+        'polymorphic_identity': 'task',
+        'polymorphic_on': type
+    }
+
     tags = relationship(
         'Tag',
         back_populates='tasks',
         secondary=task_tag_association_table
     )
 
-    def __init__(
-        self,
-        id,
-        title,
-        agile=None,
-        closed=None,
-        due=None,
-        wait=None,
-        body=None,
-        estimate=None,
-        fun=None,
-        priority=None,
-        state=None,
-        value=None,
-        willpower=None,
-    ):
-        self.id = id
-        self.agile = agile
-        self.closed = closed
-        self.due = due
-        self.wait = wait
-        self.title = title
-        self.estimate = estimate
-        self.body = body
-        self.fun = fun
-        self.priority = priority
-        self.state = state
-        self.value = value
-        self.willpower = willpower
+
+class RecurrentTask(Task):
+    __tablename__ = 'recurrent_task'
+    id = Column(String, ForeignKey('task.id'), primary_key=True)
+    recurrence_type = Column(
```

Comment:
Despite being a string, is being shown as an object in the `frozen` report.

`pydo frozen` returns:

```bash
ID    Title    Recurrence        Recurrence Type
----  -------  ----------------  --------------------------
s     test     2020-05-07 00:00  01E7K2H2G0ZRMMGG2TNAAAAAAA
```

Reply:
Fixed in 90b49f7, I tried to do the test, but it wasn't trivial.

Reply:
Actually it was, b8c83a4

Reply:
But keep in mind that you have frozen the child task and not the parent, so it won't stop the recurrence.

Although it's interesting to be able to freeze `Task` tasks, in case for example you want to mark them as the bottom of the backlog and not shown.

Reply:
Ok, now I see that a similar thing happens with the `Parent` attribute:

```
ID    Title    Recurrence    Recurrence Type    Due               Parent
----  -------  ------------  -----------------  ----------------  --------------------------
s     test                                      2020-05-07 00:00  01E7K2H2G0ZRMMGG2TNAAAAAAA
```

Reply:
That is expected, it's the ulid of the parent task so you can act on it